rotkehlchen/history/events.py

```diff
@@ -365,16 +349,31 @@ def fail_history_cb(error_msg: str) -> None:
         self.eth_tx_decoder.get_and_decode_undecoded_transactions(limit=None)
         step = self._increase_progress(step, total_steps)
 
-        # Include all external trades and trades from external exchanges
-        for location in EXTERNAL_LOCATION:
-            self.processing_state_name = f'Querying {location} trades history'
-            with self.db.conn.read_ctx() as cursor:
-                external_trades = self.db.get_trades(
-                    cursor,
-                    filter_query=TradesFilterQuery.make(location=location),
-                    has_premium=True,  # we need all trades for accounting -- limit happens later
-                )
-            history.extend(external_trades)
+        # Include all trades
+        self.processing_state_name = 'Querying trades history'
+        with self.db.conn.read_ctx() as cursor:
+            trades = self.db.get_trades(
```

Comment:
I believe there is no reason to have them here. Can you move this db queries just after querying the exchanges? I found it confusing since they are related things but they were distant in the code. Also this queries now query both excchange trades and external locations trades, can you write that in one comment?